registries/_format/decimal128.md

```diff
@@ -0,0 +1,27 @@
+---
+owner:
+issue:
+description: A decimal floating-point number with 34 significant decimal digits
+base_type: string number
```

Comment:
No, it's a well-known problem for some serializers and parsers, and already called out in https://www.rfc-editor.org/rfc/rfc8259#section-6

Reply:
@ralfhandl is this `string, number` as in you can use either, or do you mean `string number` as in "a string containing a number"?  If the latter, I would just say `string`.  If the former, I suggest the comma for clarity 🧐

Reply:
this is a good callout for clarity. If we change this, we should also change it in the infamous decimal format

Reply:
@ralfhandl Are you referring to this line:

> implementations will approximate JSON numbers within the expected precision

I don't think that RFC 8259 wording is particularly clear about potential loss of precision when storing decimals as numbers.

Reply:
@darrelmiller @ralfhandl we would probably want to reference https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00#section-4.2.1 which includes:

>  Whitespace and formatting concerns, including different lexical
   representations of numbers that are equal within the data model, are
   thus outside the scope of JSON Schema.  JSON Schema vocabularies
   ([Section 8.1](https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00#section-8.1)) that wish to work with such differences in lexical
   representations SHOULD define keywords to precisely interpret
   formatted strings within the data model rather than relying on having
   the original JSON representation Unicode characters available.

Reply:
@handrews It's `"type": ["string", "number"]`, we use content negotiation to allow clients select whether the server will send JSON numbers or wrap the numbers as JSON strings.